README.md

```diff
@@ -245,4 +245,4 @@ $ yarn test:unit
 
 ## License
 
-Licensed under GNU General Public License (GPL), version 3 or later.
+Licensed under GNU General Public License (GPL), version 3 or later.
```

Comment:
Why does this line change?

Reply:
I frankly have no idea why this is showing as changed.

Reply:
i think it must be because the added tag doesn't count as an additional line, like total number of lines didn't change according to github but the added does increase the number of lines by one.